docs/guides/transactions/running-background-jobs.md

```diff
@@ -118,20 +43,87 @@ namespace StarcounterApplication4
 }
 ```
 
-**Note:** using `thread.Start()` instead of `ThreadPool.QueueUserWorkItem` will lead to the following entries in the Starcounter log and the shutdown will take longer time.
+## Basic Information About Scheduling
+
+Each Starcounter scheduler has a queue of tasks that are supposed to be run on this scheduler. Tasks are picked from the queue and executed. To put a task in a queue, the `Scheduling.RunTask` should be used. When scheduling a task, you can specify the scheduler number, and if the thread should wait for the task to be picked by scheduler and completed. Here is the signature of the `Scheduling.RunTask`:
```

Comment:
Great explanation! One phrasing point: I would write "To put a task in a queue, use `Scheduling.RunTask`". This makes the sentence active and less wordy.

Also, "to be picked by **the** scheduler"

Reply:
>and if the thread should wait for the task to be picked by scheduler and completed

really?